examples/combobox/js/listbox.js

```diff
@@ -7,7 +7,7 @@ var Listbox = function (domNode, comboboxObj) {
     msgPrefix = 'Listbox constructor argument domNode ';
 
   // Check whether domNode is a DOM element
-  if (!(domNode instanceof Element)) {
+  if (!domNode instanceof Element) {
```

Comment:
```suggestion
  if (!(domNode instanceof Element)) {
```
This needs the inner parantheses, since `!` binds more strongly than `instanceof`. That is, the above is equivalent to

```
if ((!domNode) instanceof Element) {
```

Reply:
I noticed now that this also fails in the linter:
```
/home/travis/build/w3c/aria-practices/examples/combobox/js/listbox.js
  10:7  error  Unexpected negating the left operand of 'instanceof' operator  no-unsafe-negation
```

Reply:
Fixed in https://github.com/w3c/aria-practices/pull/1255/commits/29e94eee65a532be3dcbbf0e4c9bad5083584454